website/blog/2018-07-27-removing-babels-stage-presets.md

```diff
@@ -186,4 +186,10 @@ Who is doing that work, and is it our responsibility to make sure everything wor
 
 The purpose of this project is to act as a part of the TC39 process: being a resource for both implementing newer (Stage 0-2) proposals and receiving feedback from users while also supporting older versions of JavaScript. We hope this post sheds more light on how we are trying, as best we can, to better align this project in the JavaScript ecosystem.
 
+---
+
+If you appreciate this post and the work we're doing on Babel, you can support me on [Patreon](https://www.patreon.com/henryzhu), ask your company to sponsor us on [Open Collective](https://opencollective.com/babel), or better yet get involved with Babel as part of your job/work. We'd appreciate the collective ownership.
+
+With thanks to all the reviewers (too many to list)!
```

Comment:
You could link to this PR. That would be like "behind the scene" which could attract peoples.